evennia/utils/containers.py

```diff
@@ -202,14 +202,17 @@ def load_data(self):
         """
         if self.typeclass_storage is None:
             self.typeclass_storage = {}
-            for key, data in self.loaded_data.items():
+            for key, data in list(self.loaded_data.items()):
                 try:
                     typeclass = data.get("typeclass", settings.BASE_SCRIPT_TYPECLASS)
                     self.typeclass_storage[key] = class_from_module(typeclass)
                 except Exception:
-                    logger.log_trace(
-                        f"GlobalScriptContainer could not start import global script {key}."
+                    logger.log_err(
```

Comment:
It sounds like this should still use `log_trace` in order to properly log the exception that it hits to get to this point

Reply:
Reverted it. 
My bad, I thought trace was a lower than DEBUG log level instead of a "traceback/exception" log level.